src/axom/primal/geometry/BezierPatch.hpp

```diff
@@ -777,6 +799,581 @@ class BezierPatch
     }
   }
 
+  void evaluate_first_derivatives(T u,
```

Comment:
Please add doxygen comments for the new methods

Reply:
Doxygen comments should all in there now, sorry for making you wait so long!